ts/parts-more/PackedBubbleSeries.ts

```diff
@@ -10,15 +10,33 @@
 
 'use strict';
 
+import Chart from '../parts/Chart.js';
+import Color from '../parts/Color.js';
+const color = Color.parse;
 import H from '../parts/Globals.js';
+import Point from '../parts/Point.js';
+import U from '../parts/Utilities.js';
+const {
+    addEvent,
+    clamp,
+    defined,
+    extend,
+    extendClass,
+    fireEvent,
+    isArray,
+    isNumber,
+    merge,
+    pick,
+    seriesType
+} = U;
 
 /**
  * Internal types
  * @private
  */
 declare global {
     namespace Highcharts {
-        interface Chart {
+        interface ChartLike {
```

Comment:
Is it a proper name? Sounds a little strange...

Reply:
This has to change that way, otherwise you will not be able to reference the ES6-imported Chart type in the internal namespace.

Reply:
Also: The internal namespace is a temporary one to help during the migration. You should not rely on it and always favor ES6 imports. A good example in how to do that is the accessibility code.